ooiservices/app/uframe/controller.py

```diff
@@ -184,36 +175,6 @@ def get_uframe_platforms(mooring):
     except:
         return internal_server_error('uframe connection cannot be made.')
 
-@auth.login_required
-@api.route('/get_glider_track/<string:ref>')
-def get_uframe_glider_track(ref):
-    '''
-    Given a reference designator, returns a GeoJSON LineString for glider
-    tracks
-    '''
-    # we will always want the telemetered data, and the engineering stream
-    # data should reside in the same place
-    res = get_json('telemetered_glider_eng_telemetered', ref)
-    try:
-        if res.status_code == 200:
-            res_arr = json.loads(res.data)['data']
-            # load the JSON into a shapely LineString.
-            track = LineString([(pt['m_gps_lon'], pt['m_gps_lat'])
-                                for pt in res_arr if pt['m_gps_lon'] != 'NaN'
-                                and pt['m_gps_lat'] != 'NaN'])
-            # serialize the Python object of containing tracks to GeoJSON
-            return Response(json.dumps(track),
-                            mimetype='application/json')
-        else:
-            # if not a valid response, attempt to return the response as is.
-            return Response(json.dumps({'type': "LineString",'coordinates':[],'note':'invalid status code'}),
-                            mimetype='application/json')
-            #return res.text, res.status_code, res.headers.items()
-    except AttributeError:
-        #return nothing
-        return Response(json.dumps({'type': "LineString",'coordinates':[],'note':'AttributeError'}),
-                            mimetype='application/json')
-
```

Comment:
Why is this chunk gone?  Does cutting this make us no longer support glider tracks?